### DISCUSSION TITLE
When will 3.3.7 be officially released?

### DISCUSSION THREAD

**Discussion by mlnl221** [nicotine-plus/nicotine-plus#3211] [General]
Im waiting on 3.3.7 for some bug fixes but im using a docker container version that pulls only the latest release (3.3.6), when is 3.3.7 planned to be release?

**Comment by mathiascode** [comment]
It was planned at the beginning of the month, but had to be postponed due to regressions on Windows. Maybe in a few days if nothing else goes wrong.

**Comment by mlnl221** [comment]
Shamelss plug: For anyone that uses gluetun and proton VPN, I have written a nicotine+ script that only works on 3.3.7+ that automatically updates your port to the VPNs port (due to gluetun disconnected and reconnecting)

https://github.com/minicoz/nicotine-plus-gluetun-auto-port-set

**Comment by mathiascode** [comment]
3.3.7 has been released. https://nicotine-plus.org/doc/DOWNLOADS.html
